src/CMakeLists.txt

```diff
@@ -1,39 +1,3 @@
-set(CMAKE_C_FLAGS_DEBUG "-DUSE_ASSERT_CHECKING=1")
```

Comment:
I don't think this separation into a separate file is necessary anymore since test/src is included from this CMakeList now (via `add_subdirectory(../test/src "${CMAKE_CURRENT_BINARY_DIR}/test/src")`) and thus should inherit all these values